nlptest/transform/robustness.py

```diff
@@ -88,79 +88,103 @@ async def async_run(cls, sample_list: List[Sample], model: ModelFactory, **kwarg
             cls.run(sample_list, model, **kwargs))
         return created_task
 
-
-
 class UpperCase(BaseRobustness):
     alias_name = "uppercase"
 
     @staticmethod
-    def transform(sample_list: List[Sample]) -> List[Sample]:
-        """Transform a list of strings with uppercase robustness
+    def transform(sample_list: List[Sample], prob: float = None) -> List[Sample]:
+        """Transform a list of strings with uppercase robustness.
```

Comment:
By setting the default value to `1.0` you can directly have `num_transform_words = int(prob * len(words))` and don't need to check if `prob is None` 😃

Reply:
ya sure

Reply:
Alright !